README.md

```diff
@@ -0,0 +1,165 @@
+### Real User Monitoring (Browser Implementation)
```

Comment:
In Markdown, typically begin with H1 (e.g., `#`)... think of them as logically nested sections...

So please format this as something like:

```
# Real User Monitoring

## Browser Instrumentation

### Overview

### Setup 

#### For HTML and Javascript based application

#### For NextJS

## NodeJS Instrumentation

### Overview

### Setup
```

This should also allow you to remove the `=========` which seem to overrun in my browser.